src/sentry/integrations/discord/client.py

```diff
@@ -15,7 +15,10 @@ class DiscordClient(IntegrationProxyClient):
     base_url: str = "https://discord.com/api/v10"
 
     # https://discord.com/developers/docs/resources/guild#get-guild
-    get_guild_url = "/guilds/{guild_id}"
+    guild_url = "/guilds/{guild_id}"
```

Comment:
Naming constants with uppercase letters is a great practice since it makes them stand out, so `GUILD_URL` and `USERS_GUILD_URL`